resources/testdata/cf-sample-app-nodejs.yaml

```diff
@@ -0,0 +1,9 @@
+# From https://github.com/cloudfoundry-samples/cf-sample-app-nodejs/blob/master/manifest.yml
```

Comment:
the content of the file is a bit different for the linked one. Maybe we can add a better description?

Reply:
Yeah, I found a pending PR to fix it:
https://github.com/cloudfoundry-samples/cf-sample-app-nodejs/pull/10/files

I can amend the comment to reference the PR instead. WDYT?

Reply:
ah ok, now i get it.
you can add the link of the PR or amending the link. not a strong opinion.

Reply:
Updated and added 2 new examples that use buildpacks